Source/Hammer.spoon/init.lua

```diff
@@ -0,0 +1,95 @@
+--- === Hammer ===
```

Comment:
I think I want to try and push against Spoons being collections of things - that gets us back to the problem we have now with everyone's `init.lua` being a big mash of things. If the Spoons stay fairly close to the UNIX philosophy, it ought to be easier for users to select exactly the things they want.